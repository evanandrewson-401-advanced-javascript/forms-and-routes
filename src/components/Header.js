import React from 'react';

const Header = () => {
  return (
    <header>
      <h1>Avatar characters</h1>
    </header>
  )
}

export default Header;