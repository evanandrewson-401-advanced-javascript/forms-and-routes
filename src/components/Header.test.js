import Header from './Header';
import React from 'react';
import { shallow } from 'enzyme';

describe('Header component', () => {
  it('renders a header', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
  });
});
