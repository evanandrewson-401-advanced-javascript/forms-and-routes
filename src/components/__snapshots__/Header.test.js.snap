// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component renders a header 1`] = `
<header>
  <h1>
    Avatar characters
  </h1>
  <nav>
    <Link
      to="/"
    >
      Home
    </Link>
    <Link
      to="/list"
    >
      List of Characters
    </Link>
  </nav>
</header>
`;
